sdk/search/Azure.Search.Documents/tests/Utilities/TestExtensions.cs

```diff
@@ -161,49 +156,40 @@ public static string GetName(this Random random, int length = 8)
         /// <param name="latitude">The latitude.</param>
         /// <param name="longitude">The longitude.</param>
         /// <returns></returns>
-#if EXPERIMENTAL_SPATIAL
         public static GeoPoint CreatePoint(double longitude, double latitude) =>
             new GeoPoint(new GeoPosition(longitude, latitude));
 
         public static GeoPoint CreateDynamicPoint(double longitude, double latitude) =>
             CreatePoint(longitude, latitude);
-#else
-        public static GeographyPoint CreatePoint(double longitude, double latitude) =>
-            // Note: GeographyPoint takes latitude first, unlike GeoPoint
-            GeographyPoint.Create(latitude, longitude);
 
-        public static SearchDocument CreateDynamicPoint(double longitude, double latitude) =>
-            CreatePoint(longitude, latitude).AsDocument();
-#endif
-
-        /// <summary>
-        /// Converts the <see cref="GeographyPoint"/> to a <see cref="SearchDocument"/>.
-        /// </summary>
-        /// <param name="value">The <see cref="GeographyPoint"/> to convert.</param>
-        /// <returns>A <see cref="SearchDocument"/> for the given <paramref name="value"/>.</returns>
-        public static SearchDocument AsDocument(this GeographyPoint value)
-        {
-            if (value == null)
-            {
-                return null;
-            }
-
-            List<double> coords = new List<double>
-            {
-                value.Longitude,
-                value.Latitude,
-            };
-
-            if (value.Z != null)
-            {
-                coords.Add(value.Z.Value);
-            }
-
-            return new SearchDocument()
-            {
-                ["type"] = "Point",
-                ["coordinates"] = coords.ToArray()
-            };
-        }
+        ///// <summary>
+        ///// Converts the <see cref="GeographyPoint"/> to a <see cref="SearchDocument"/>.
+        ///// </summary>
+        ///// <param name="value">The <see cref="GeographyPoint"/> to convert.</param>
+        ///// <returns>A <see cref="SearchDocument"/> for the given <paramref name="value"/>.</returns>
+        //public static SearchDocument AsDocument(this GeographyPoint value)
```

Comment:
Nit: could just clean up the code to remove these since it's all in source control.

But, this isn't used anymore? Interesting.

Reply:
I have uncommented and fixed the code. I did not see it being used anywhere.